contract/AElf.Contracts.Genesis/BasicContractZero.cs

```diff
@@ -160,7 +172,18 @@ public override Hash ProposeUpdateContract(ContractUpdateInput input)
         Assert(info != null, "Contract not found.");
         AssertAuthorityByContractInfo(info, Context.Sender);
         AssertContractVersion(info.ContractVersion, input.Code, info.Category);
-        AssertContractExists(HashHelper.ComputeFrom(input.Code.ToByteArray()));
+
+        var codeHash = HashHelper.ComputeFrom(input.Code.ToByteArray());
+        AssertContractExists(codeHash);
```

Comment:
Understand this is a legacy method, but it's was named wrongly. Can we fix it and change it to `AssertContractNotExists`?